examples/amp-story/branching.html

```diff
@@ -0,0 +1,78 @@
+<!doctype html>
+<html amp lang="en">
+  <head>
+    <meta charset="utf-8">
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script async custom-element="amp-story"
+        src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+    <script async custom-element="amp-sidebar" src="https://cdn.ampproject.org/v0/amp-sidebar-0.1.js"></script>
+    <title>amp-story branching</title>
+    <meta name="viewport"
+          content="width=device-width,minimum-scale=1,initial-scale=1">
+    <link rel="canonical" href="./branching.html">
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <style amp-custom>
+      amp-story-page {
+        background: white;
+        color: rgba(0, 0, 0, 0.87);
+        font-family: sans-serif;
+      }
+      h1.hello-world {
+        color: white;
+        text-shadow: 3px 3px rgba(0, 0, 0, 0.5);
+      }
+      amp-sidebar {
+        font-size: 18px;
+        background-color: #fff;
+      }
+      amp-sidebar div {
+        border: 3px solid green;
+        height: 100vh;
+      }
+    </style>
+  </head>
+
+  <body>
+    <amp-story id="story" standalone publisher="AMP Team" title="Visual Diff Test"
+        publisher-logo-src="/examples/visual-tests/photos/120.png"
+        poster-portrait-src="/examples/visual-tests/picsum.photos/image981_900x1600.jpg"
+        poster-landscape-src="/examples/visual-tests/picsum.photos/image981_1600x900.jpg"
+        poster-square-src="/examples/visual-tests/picsum.photos/image981_1600x1600.jpg">
+      <amp-sidebar id="sidebar1" layout="nodisplay" >
+        <button id="close-button" on="tap:sidebar1.close">Close</button>
+        <ul>
+          <li><span><a href="/examples/amp-story/branching.html#page=page-2&foo=bar">Absolute fragment link</a></span></li>
+          <li><span><a href="#page=page-2">relative fragment link</a></span></li>
+          <li><a href="#page=page-3&foo=bar">In Link Hash</a></li>
+          <li><a href="/examples/amp-story/rtl.html#page=page-2">Link to another story using fragment</a></li>
+          <li on="tap:story.goToPage(id=page-3)">In Link Action</li>
```

Comment:
Does this link work without the experiment enabled?

Reply:
Nope, I forgot to remove it. Removed it now.